testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/certs/ocsp/TestingOcspServer.java

```diff
@@ -123,12 +126,21 @@ public void start() throws Exception {
         server = new ClientAndServer(port);
         server.when(
                 request()
-                        .withMethod("POST")
-                        .withPath("/ocsp"),
+                        .withMethod(this.method)
```

Comment:
Instead of configuring the expected method based on the JDK version, it's possible to configure the mock server with multiple expectations, e.g.,

```
server.when(request().withMethod("POST")...)...respond();
server.when(request().withMethod("GET")...)...respond();
```
That way, we'll be able to handle the correct method regardless of the JDK version without having to worry about detecting the version ourselves.

Reply:
Fixed!